Disable Fortran ERI tests as those have very strict error tolerances. This patch is updated from Libint-2.6.0_remove-test-eri.patch
See https://github.com/evaleev/libint/issues/188
author: Ben Czaja (SURF) based on orginal patch from Alex Domingo (Vrije Universiteit Brussel)
diff -W 152 -Nru libint-2.7.2-orig/export/cmake/CMakeLists.txt.export libint-2.7.2/export/cmake/CMakeLists.txt.export
--- libint-2.7.2-orig/export/cmake/CMakeLists.txt.export        2022-06-20 07:51:55.000000000 +0200
+++ libint-2.7.2/export/cmake/CMakeLists.txt.export     2022-08-16 17:38:22.347117248 +0200
@@ -443,7 +443,7 @@
             PROPERTIES FIXTURES_REQUIRED LIBINT2_FORTRAN_EXAMPLE_EXEC)

     if (LIBINT_HAS_CXX_API)
-        add_executable(fortran_test-libint2 EXCLUDE_FROM_ALL fortran/test.cc fortran/test-eri.cc $<TARGET_OBJECTS:libint_f>)
+        add_executable(fortran_test-libint2 EXCLUDE_FROM_ALL fortran/test.cc $<TARGET_OBJECTS:libint_f>)
         target_link_libraries(fortran_test-libint2 libint2_cxx libint_f)
         target_include_directories(fortran_test-libint2 PRIVATE $<TARGET_PROPERTY:libint_f,Fortran_MODULE_DIRECTORY>)
         add_test(libint2/fortran_test/build "${CMAKE_COMMAND}" --build ${PROJECT_BINARY_DIR} --target fortran_test-libint2)
